ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -348,6 +393,12 @@ class FamilyReads extends React.PureComponent {
                 />
                 {sampleTypes.some(sampleType => RNA_TRACK_TYPE_LOOKUP.has(sampleType)) && (
                   <div>
+                    <Divider horizontal>Range</Divider>
```

Comment:
Per the ticket: This option will be available for all data types.
This means its always shown, not just if an RNA seq type is shown